net/ovsd/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ovsd
+PKG_VERSION:=2020-10-12
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.hhi.fraunhofer.de/wn-ina/ovsd.git
+PKG_SOURCE_VERSION:=e6a07f27dd8a5893eb6110c69922077995b9b26a
+PKG_MIRROR_HASH:=9d903b3fecd8a3e102e1477efe479fc3851b1814ea0852e1dae39065b88cd157
+
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+
+PKG_MAINTAINER:=Arne Kappen <akappen@inet.tu-berlin.de>
```

Comment:
Ack from the maintainer?

Reply:
Yes, he was waiting for the needed changes in netifd to be merged which was only feasible after 21.02 branched off.